### ISSUE TITLE
Vendored HPy includes pre-built static libraries

### ISSUE THREAD

**Issue by stefanor** [pypy/pypy#4871]
I assume these shouldn't have been vendored, they are x86_64 static libs:

```
./lib_pypy/hpy/devel/lib/cpython/libhpy-ctx-cpython.a
./lib_pypy/hpy/devel/lib/hybrid/libhpy-extra-hybrid.a
./lib_pypy/hpy/devel/lib/universal/libhpy-extra-universal.a
./pypy/module/_hpy_universal/_vendored/hpy/devel/lib/cpython/libhpy-ctx-cpython.a
./pypy/module/_hpy_universal/_vendored/hpy/devel/lib/hybrid/libhpy-extra-hybrid.a
./pypy/module/_hpy_universal/_vendored/hpy/devel/lib/universal/libhpy-extra-universal.a
```

**Comment by mattip** [comment]
That is a mistake. Does it break debian builds?

**Comment by stefanor** [comment]
> Does it break debian builds?

No, it doesn't break anything. But our linting tools flagged them, because they weren't named in a way that would avoid collisions between architectures (if they were built at build time).

**Comment by mattip** [comment]
Cool, thanks for reporting.

**Comment by mattip** [comment]
Closed by #4872
